README.md

```diff
@@ -1,3 +1,45 @@
-# Repository for generating Gwent card images.
+# Gwent-data Card Generator 1.2
+This PHP script reads the content of the cards.json file generated by **Gwent Data** and generate cards images to be used with Gwent tools.
 
-Placeholder README
+## Requirement
+The script needs PHP >= 5.6 installed with Imagick and Imagick PHP extension installed. It can be called using HTTP or command line.
+
+The Gwent cards artworks need to be extracted from the game using the *high* card size (located in the `high.standard` game file)  with their original filenames. The [Unity Assets Bundle Extractor](https://github.com/DerPopo/UABE) can be used for this.
+
+## Installation
+1. If you're not familiar with `composer`, [download it](https://getcomposer.org/) and [install the dependencies](https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies) of the script.
+2. Copy the cards artworks into the `assets/artworks/` folder.
+3. In the `card_generator.php` file, change the value of :
+  - `VERSION` to get the right Gwent version
+  - `MAX_CARDS` to specify how many cards should be generated to the max
+4. Copy the last `cards.json` generated file in the same folder as `card_generator.php`
```

Comment:
can this also be a cli param?

Reply:
It can be. I'll remove it as constants and change it as class vars so it still has a default value.